apis/run/v1alpha3/tanzuclusterbootstraptemplate_types.go

```diff
@@ -54,6 +56,15 @@ type ValuesFrom struct {
 	ProviderRef *corev1.TypedLocalObjectReference `json:"providerRef,omitempty"`
 }
 
+type ProxyConfig struct {
+	// +optional
+	HTTPProxy string `json:"http_proxy"`
+	// +optional
+	HTTPSProxy string `json:"https_proxy"`
+	// +optional
+	NoProxy string `json:"no_proxy"`
```

Comment:
Question: Should `NoProxy` be a list of strings?

Reply:
I think it's a comma delimited string
```
# A comma delimited list of domain names which kapp controller should
  # bypass the proxy for when making requests (optional)
  noProxy: "github.com,docker.io"
```

Reply:
Saw the [link](https://gitlab.eng.vmware.com/core-build/guest-cluster-controller/-/blob/master/apis/run.tanzu/v1alpha2/tanzukubernetescluster_types.go#L267), will change to string array.